src/components/sbb-journey-summary/sbb-journey-summary.tsx

```diff
@@ -0,0 +1,157 @@
+import { Component, h, JSX, Prop } from '@stencil/core';
+import { InterfaceJourneySummaryAttributes } from './sbb-journey-summary.custom';
+import isTomorrow from 'date-fns/isTomorrow';
+import { isToday, parseISO, format, intervalToDuration, isValid } from 'date-fns';
```

Comment:
Shouldn't dates and times show the foreign time zone? If so, probably gonna need 'date-fns-tz'?